modules/calypsoify/class.jetpack-calypsoify.php

```diff
@@ -274,13 +280,15 @@ public function enqueue_for_gutenberg() {
 		);
 	}
 
-	public function insert_sidebar_html() { ?>
+	public function insert_sidebar_html() { 
+		$heading = ( $_GET[ 'post_type' ] == 'feedback' ) ? 'Feedback' : 'Plugins';
+		?>
 		<a href="<?php echo esc_url( 'https://wordpress.com/stats/day/' . Jetpack::build_raw_urls( home_url() ) ); ?>" id="calypso-sidebar-header">
 			<svg class="gridicon gridicons-chevron-left" height="24" width="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><g><path d="M14 20l-8-8 8-8 1.414 1.414L8.828 12l6.586 6.586"></path></g></svg>
 
 			<ul>
 				<li id="calypso-sitename"><?php bloginfo( 'name' ); ?></li>
-				<li id="calypso-plugins"><?php esc_html_e( 'Plugins' ); ?></li>
+				<li id="calypso-plugins"><?php esc_html_e( $heading ); ?></li>
```

Comment:
`esc_html_e` expects a textdomain. Since we translate those strings earlier, we can just echo the variable here without translating it.

```suggestion
				<li id="calypso-plugins"><?php echo esc_html( $heading ); ?></li>
```